sendgrid_helper.py

```diff
@@ -0,0 +1,22 @@
+import os
+from flask import jsonify
+from sendgrid import SendGridAPIClient
+from sendgrid.helpers.mail import Mail
+
+def send_reset_password_mail(email, token):
+    server_link = os.environ.get('SENDGRID_SERVER_LINK')
+    from_email = os.environ.get('SENDGRID_EMAIL')
+    to_email = email
+    subject = 'Reset password link'
+    content = 'To reset your password'
+    html_content = 'To reset your password <a href="%s/reset-password/%s">click here</a>' % (server_link, token)
+    message = Mail(from_email, to_email, subject, content, html_content)
+    try:
```

Comment:
try catch implementation is faulty - parent function will not know that something went wrong

Reply:
good way to fix it: catch sendgrid specific error, and raise an error that your parent function will understand; in parent function, catch it and throw 500 (only if parent is a resource function);